Client/Frontend/BraveVPN/BraveVPN.swift

```diff
@@ -33,9 +40,13 @@ class BraveVPN {
       if let error = error {
         logAndStoreError("Failed to load vpn conection: \(error)")
       }
-
-      GRDGatewayAPI.shared()._loadCredentialsFromKeychain()
-
+      
+      GRDSubscriptionManager.setIsPayingUser(true)
```

Comment:
Is this intentional? You should only mark users as being paying users if the receipt includes a valid subscription line item